src/pages/lib/auth/credential_storage/q/platform/[platform].mdx

```diff
@@ -0,0 +1,10 @@
+export const meta = {
+  title: `Credential storage`,
+  description: `Learn how to customize credential storage.`,
+};
+
+The Amplify Auth category persists authentication-related information to make it available to other Amplify categories and to your application.
```

Comment:
JS has this: https://docs.amplify.aws/lib/auth/manageusers/q/platform/js/#managing-security-tokens

Don't know how important it is to match structure with them. Should it go with `Accessing credentials` and/or have that renamed to `Managing credentials`?

Reply:
I think having a seperate sidebar item is helpful, personally, but I don't feel strongly about it.  In any case, I've changed the title to `Managing credentials`.